components/root/root.jsx

```diff
@@ -146,8 +147,74 @@ export default class Root extends React.Component {
         };
     }
 
+    getSidebarCategories = () => {
+        const sidebarCategories = [
+            document.getElementById('unreadsChannelList'),
+            document.getElementById('favoriteChannelList'),
+            document.getElementById('publicChannelList'),
+            document.getElementById('privateChannelList'),
+            document.getElementById('directChannelList'),
+        ].filter((element) => Boolean(element));
+
+        return sidebarCategories;
+    }
+
     handleAccessibilityKeys = (e) => {
-        if (isKeyPressed(e, Constants.KeyCodes.TAB)) {
+        if (isKeyPressed(e, Constants.KeyCodes.DOWN)) {
+            const lhsList = document.getElementById('lhsList');
+            const sidebarCategories = this.getSidebarCategories();
+            if (lhsList.contains(e.target)) {
+                const containsElement = (element, i, elementsList) => {
+                    if (element.contains(e.target)) {
+                        this.currentSidebarFocus = i + 1;
+                        return true;
+                    } else if (element.contains(e.target) && i === elementsList.length - 1) {
+                        this.currentSidebarFocus++;
+                        return false;
+                    }
+                    return false;
+                };
+
+                if (!sidebarCategories.some(containsElement)) {
+                    this.currentSidebarFocus = 0;
+                }
+
+                if (this.currentSidebarFocus === sidebarCategories.length) {
+                    this.currentSidebarFocus = 0;
+                }
+
+                const activeElement = sidebarCategories[this.currentSidebarFocus];
+                this.addFocusClass(activeElement);
+                this.currentSidebarFocus++;
+            }
+        } else if (isKeyPressed(e, Constants.KeyCodes.UP)) {
```

Comment:
Doesn't this conflict with another keyboard shortcut we have for UP that opens the last post for editing? Have you tested that?

Reply:
Took a look at the ticket and it looks like this should only happen if the sidebar region is already focused so I think you'll need to add a check for that to make sure it doesn't conflict with other keyboard shortcuts

Reply:
The check is already there, which is:
`if (lhsList.contains(e.target)) {`

Reply:
So it won't conflict with that as this checks whether the focus is in the sidebar. This does not even work even if the user is in the sidebar using (alt + up/down) shortcuts, as those do not focus the fields.

It only works if the focus exists within the lhs list.

Reply:
Ok cool 👍